index_test.go

```diff
@@ -1775,7 +1775,7 @@ func TestListRepos(t *testing.T) {
 			Stats: RepoStats{
 				Shards:                     1,
 				Documents:                  4,
-				IndexBytes:                 300,
+				IndexBytes:                 412,
```

Comment:
why did this go up?

Reply:
combinedNgramOffset.SizeBytes() didn't account for header bytes (slice headers mostly), has a compressed representation for ascii, and has a lower overhead for low ngram counts (like here in the test case). An empty `btreeIndex` already uses 92 bytes. However, the cost amortizes once the tree fills up. Even small repos have a fulll b-tree with ~1000 leaf nodes.